app/Http/Controllers/ReadingItemController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\ReadingItem;
+use Illuminate\Http\Request;
+
+class ReadingItemController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('reading-items.index')->with([
+            'reading_items' => ReadingItem::all()->sortByDesc('created_at'),
```

Comment:
What does abstract *ReadingItem* signify in the real world. @rathorevaibhav

Reply:
It's basically the article/read people will share via the chrome plugin. Has the following attributes:
1. Description
2. URL
3. Recommended_by

I feel that the name is a bit off. Any suggestions for a better name of this entity?

Reply:
I think it's fine until that's the closest representation in the real world.